.github/ISSUE_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 <!-- Thanks for filing an issue! Before hitting the button, please answer these questions.-->
 
-**Is this a request for help?** (If yes, you should use our troubleshooting guide and community support channels, see http://kubernetes.io/docs/troubleshooting/.):
+**Is this a request for help?** (If yes, you should use our troubleshooting guide and community support channels, see https://kubernetes.io/docs/tasks/debug-application-cluster/troubleshooting/.):
```

Comment:
@supereagle Thanks for your review! Since this is just a nit and Christoph has already LGTM'd and all the checks passed, let's leave it as is :)